src/app/(proper_react)/(redesign)/(authenticated)/admin/announcements/AnnouncementsAdmin.tsx

```diff
@@ -156,315 +159,299 @@ export const AnnouncementsAdmin = (props: Props) => {
+        >
+          Announcements
+        </button>
+        <button
+          className={`${styles.tabButton} ${
+            activeTab === "strings" ? styles.activeTab : ""
+          }`}
+          onClick={() => setActiveTab("strings")}
+        >
+          Fluent Strings
+        </button>
+        <button
+          className={`${styles.tabButton} ${
+            activeTab === "docs" ? styles.activeTab : ""
+          }`}
+          onClick={() => setActiveTab("docs")}
+        >
+          Docs
+        </button>
+      </div>
+      {/* <View /> */}
+      {activeTab === "announcements" && (
+        <>
+          {/* Announcements Block */}
+          <div className={styles.wrapper}>
+            {/* List of announcements */}
+            <div className={styles.notificationsWrapper}>
+              <h1>All Announcements</h1>
+              <ul>
+                {sortedAnnouncements.map((notification) => (
+                  <li
+                    key={notification.id}
+                    className={
+                      activeAnnouncementId === notification.id
+                        ? styles.active
+                        : ""
+                    }
+                    onClick={() => handleClick(notification.id)}
+                  >
+                    <div>
+                      <p className={styles.title}>
+                        <span className={styles.missingLabelContainer}>
+                          <LocalizedAnnouncementString
+                            announcement={notification}
+                            type="title"
+                          />
+                          <MissingFluentId
+                            announcement={notification}
+                            type="title"
+                          />
+                        </span>
+                      </p>
+                      <p className={styles.description}>
+                        <span className={styles.missingLabelContainer}>
+                          <LocalizedAnnouncementString
+                            announcement={notification}
+                            type="description"
+                          />
+                          <MissingFluentId
+                            announcement={notification}
+                            type="description"
+                          />
+                        </span>
+                      </p>
+                    </div>
+                    <div className={styles.pills}>
+                      <div
+                        className={`${styles.statusPill} ${styles[notification.label]}`}
+                      >
+                        {notification.label}
+                      </div>
+                    </div>
+                  </li>
+                ))}
+                <button
+                  className={styles.addButton}
+                  onClick={() => setIsModalOpen(true)}
+                >
+                  + Add new announcement
+                </button>
```

Comment:
A `<button>` as a direct child of `<ul>` is not valid and would need to be wrapped in an `<li>`.